libs/sdk-backend-tiger/src/convertors/fromBackend/afm/result.ts

```diff
@@ -1,24 +1,27 @@
 // (C) 2019-2020 GoodData Corporation
+import {
+    AttributeGranularityResourceAttribute,
+    DimensionHeader,
+    ExecutionResult,
+    isResultAttributeHeader,
+} from "@gooddata/api-client-tiger";
 import {
     DataValue,
-    IResultHeader,
     IDimensionDescriptor,
-    isAttributeDescriptor,
     IDimensionItemDescriptor,
     IMeasureDescriptor,
+    IResultHeader,
+    isAttributeDescriptor,
     isMeasureGroupDescriptor,
 } from "@gooddata/sdk-backend-spi";
-import { Execution, AttributeGranularityResourceAttribute } from "@gooddata/api-client-tiger";
-import isResultAttributeHeader = Execution.isResultAttributeHeader;
-import isResultMeasureHeader = Execution.isResultMeasureHeader;
 import { DateAttributeGranularity } from "@gooddata/sdk-model";
-import { toSdkGranularity } from "../dateGranularityConversions";
-import { DateFormatter } from "../dateFormatting/types";
 import { createDateValueFormatter } from "../dateFormatting/dateValueFormatter";
+import { DateFormatter } from "../dateFormatting/types";
+import { toSdkGranularity } from "../dateGranularityConversions";
 
 export type TransformerResult = {
     readonly headerItems: IResultHeader[][][];
-    readonly data: DataValue[][] | DataValue[];
+    readonly data: DataValue[];
```

Comment:
How come the `DataValue[][]` can no longer happen?

Reply:
I'll check it again

Reply:
fixed